src/nets/e2e_asr_attctc_th.py

```diff
@@ -1897,6 +1967,212 @@ def recognize_beam(self, h, lpz, recog_args, char_list, rnnlm=None):
         # remove sos
         return nbest_hyps
 
+    def recognize_beam_batch(self, h, hlens, lpz, recog_args, char_list, rnnlm=None):
```

Comment:
Same comment.
You should replace the original `recognize_beam` to this `recognize_beam_batch`